internal/fullnode/labels.go

```diff
@@ -23,6 +24,7 @@ func defaultLabels(crd *cosmosv1.CosmosFullNode, kvPairs ...string) map[string]s
 		kube.NameLabel:       appName(crd),
 		kube.VersionLabel:    kube.ParseImageVersion(crd.Spec.PodTemplate.Image),
 		networkLabel:         crd.Spec.ChainSpec.Network,
+		typeLabel:            string(crd.Spec.Type),
```

Comment:
@nix what if Type is not set should we default to "FullNode"?

Reply:
Yes please